fe/fe-core/src/main/java/com/starrocks/server/Session.java

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.server;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+// used for temporary tables to store all temporary table ids created by a session,
+// maintain the mapping relationship of databaseId -> tableName -> tableId.
+public class Session {
```

Comment:
Session may be not a good class name now, it's looks only used on temporary table, but many feature(user variable, session variable) is session level, but doesn't implement in here, I think it's confusing.

of course, we can refactor code and move feature to here, but I think `TemporaryTableMgr` is more like `SessionMgr`?

Reply:
> Session may be not a good class name now, it's looks only used on temporary table, but many feature(user variable, session variable) is session level, but doesn't implement in here, I think it's confusing.
> 
> of course, we can refactor code and move feature to here, but I think `TemporaryTableMgr` is more like `SessionMgr`?

any suggestion about naming? @Seaven